### PR TITLE
D3ASIM-3652: Support future markets in myco sdk

### PR DESCRIPTION
**Pull request by Hassan754**

### CODE REVIEW

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Comment by Hassan754** [comment]
PS. The failing test is the codecov, for some reason this is not clearly shown

**Review by spyrostz — DISMISSED** [review]
LGTM

File: src/d3a/models/market/__init__.py [hunk 1]
```diff
@@ -135,6 +135,12 @@ def _is_constant_fees(self):
     def open_bids_and_offers(self, **kwargs) -> Tuple[List, List]:
         return list(self.bids.values()), list(self.offers.values())
 
+    def orders_per_slot(self) -> Dict[str, Dict]:
+        """Return all orders in the market per time slot."""
+        bids = [bid.serializable_dict() for bid in self.bids.values()]
+        offers = [offer.serializable_dict() for offer in self.offers.values()]
```
Comment (fievelk) [inline src/d3a/models/market/__init__.py:141]: Why are we converting with `serializable_dict()` here? Given the name of the method, I would expect to receive `Bid` and `Offer` instances.

Comment (Hassan754) [reply src/d3a/models/market/__init__.py:141]: The actual requirement is a dictionary of time_slot: serialized orders
I have already added the return type hint, however if you think that we still have to change the name of the method to reflect that, let me know

**Review by spyrostz — APPROVED** [review]
LGTM

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**